src/recipes/pumpkin-juice.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "Pumpkin Juice",
+    "description": "This is the best drink for chilly autumn days. Just grab a warm sweater and a steaming hot pumpkin juice and see the raindrops on your windows.",
+    "github": "polgarj",
+    "ingredients": [
+        {
+            "quantity": "1",
+            "measure": "tsp",
+            "ingredient": "Pumpkin pie spice"
+        },
+        {
+            "quantity": "1/2",
+            "measure": "tsp",
+            "ingredient": "vanilla"
+        },
+        {
+            "quantity": "2 1/2 ",
+            "measure": "cups",
+            "ingredient": "apple cider or apple juice"
+        },
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "water or to taste. You can add more if you are looking for something less sweet"
+        },
+        {
+            "quantity": "1",
+            "measure": "portion",
+            "ingredient": "honey or maple syrup to taste (optional. I don't add it as I find the apple juice is sweet enough on its own)"
+        }
+    ],
+    "directions": [
+        "Combine all ingredients in a blender and pulse to combine.",
+        "Simmer for 5 minutes to help enhance the flavor of the spices infuse into the mixture.",
+        "Enjoy warm or cool and enjoy chilled.",
+        "Store leftovers in an air-tight jar or container in fridge for 3-4 days. Shake or mix well before serving after storing as the pumpkin will settle."
+    ],
+    "image": "aam-panna.jpg",

```

Comment:
Please provide an image for the recipe. Existing images within the repo cannot be used.

Reply:
Sorry for the mistake. I just uploaded the image of the drink.

Reply:
Can you remove this file from the PR? 
`.history/src/recipes/pumpkin-juice_20211001204814.json`

Reply:
I removed it. Sorry again. :)

Reply:
Hey, no worries! Thanks for the contribution!